CMakeLists.txt

```
@@ -31,7 +31,7 @@ else()
 endif()
 
 # glob src
-file(GLOB_RECURSE DD_TARGET_FILES src/*.cpp)
+file(GLOB_RECURSE DD_TARGET_FILES src/*.h src/*.cpp)
```

Per https://github.com/LizardByte/libdisplaydevice/pull/10#discussion_r1567371561

We should probably not add `*.h` here.

```suggestion
file(GLOB_RECURSE DD_TARGET_FILES src/*.cpp)
include_directories("${CMAKE_SOURCE_DIR}")
```